menas/ui/service/MonitoringService.js

```diff
@@ -0,0 +1,259 @@
+    recordsStatusAggregator["Failed at conformance"].counter += +oRun["conform_records_failed"] - +oRun["std_records_failed"]
+    recordsStatusAggregator["Failed at standardization"].counter += +oRun["std_records_failed"]
+  };
+
+  this.processStandardized = function(oRun) {
+    recordsStatusAggregator["Conformed"].recordCounts.push(0)
+    recordsStatusAggregator["Failed at conformance"].recordCounts.push(0)
+    recordsStatusAggregator["Standardized (not conformed)"].recordCounts.push(+oRun["std_records_succeeded"])
+    recordsStatusAggregator["Failed at standardization"].recordCounts.push(+oRun["std_records_failed"])
+    recordsStatusAggregator["Unprocessed raw"].recordCounts.push(0)
+    recordsStatusAggregator["Inconsistent info"].recordCounts.push(0)
+
+    recordsStatusAggregator["Standardized (not conformed)"].counter += +oRun["std_records_succeeded"]
+    recordsStatusAggregator["Failed at standardization"].counter += +oRun["std_records_failed"]
+  };
+
+  this.processRaw = function(oRun) {
+    recordsStatusAggregator["Conformed"].recordCounts.push(0)
+    recordsStatusAggregator["Failed at conformance"].recordCounts.push(0)
+    recordsStatusAggregator["Standardized (not conformed)"].recordCounts.push(0)
+    recordsStatusAggregator["Failed at standardization"].recordCounts.push(0)
+    recordsStatusAggregator["Unprocessed raw"].recordCounts.push(+oRun["raw_recordcount"])
+    recordsStatusAggregator["Inconsistent info"].recordCounts.push(0)
+
+    recordsStatusAggregator["Unprocessed raw"].counter += +oRun["raw_recordcount"]
+  };
+
+  this.processInconsistent = function(oRun) {
+    recordsStatusAggregator["Conformed"].recordCounts.push(0)
+    recordsStatusAggregator["Failed at conformance"].recordCounts.push(0)
+    recordsStatusAggregator["Standardized (not conformed)"].recordCounts.push(0)
+    recordsStatusAggregator["Failed at standardization"].recordCounts.push(0)
+    recordsStatusAggregator["Unprocessed raw"].recordCounts.push(0)
+
+    // try to estimate number of records involved in this run
+    let estimatedRecordCount = 0
+    if ( oRun["raw_recordcount"] !=  undefined) {
+      estimatedRecordCount = +oRun["raw_recordcount"]
+    } else if (oRun["std_records_succeeded"] !=  undefined && oRun["std_records_failed"] !=  undefined){
+      estimatedRecordCount = (+oRun["std_records_succeeded"]) + (+oRun["std_records_failed"])
+    } else if (oRun["conform_records_succeeded"] !=  undefined && oRun["conform_records_failed"]  !=  undefined ) {
+      estimatedRecordCount = (+oRun["conform_records_succeeded"]) + (+oRun["conform_records_failed"] )
+    }
+    recordsStatusAggregator["Inconsistent info"].recordCounts.push(estimatedRecordCount)
+    recordsStatusAggregator["Inconsistent info"].counter += estimatedRecordCount
+  };
+
+
+  this.getData= function(sId, sStartDate, sEndDate) {
+    MonitoringService.clearAggregators();
+    Functions.ajax("api/monitoring/data/datasets/"
```

Comment:
I think this is the left-over way from the earlier components. 

Could you use the RestDAO.js for these calls? It'd be great to unify on one.

Reply:
Thnx for the heads up, this is a bigger one, created a new issue #473